src/navigation/NavBar.css

```diff
@@ -1,5 +1,18 @@
+.navbar-item-active {
+  border-bottom: 5px solid rgba(201, 210, 215, .4);
+}
+
+@media only screen and (min-width: 60em) {
+  .navbar-container {
+    width: 9.75rem;
```

Comment:
Another alternative for the whiteSpace is just removing the fixed width, but that also involves changes in the fixed position (will comment that soon)

Reply:
Thought about it for a while and came to the conclusion that it's probably best to keep fixed with: the sidebar menu is "structural furniture" (for want of a better term) that's at a lower level of the information architecture than the i18n. With that in mind, proposing that we bend the i18n around the sidebar rather than the other way around 😊